docs/blog/_posts/2021-04-21-scala3-rc3.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-page
+title: Scala 3.0.0-RC3 – bug fixes for 3.0.0 stable
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2021-04-21
+---
+
+Hello! With this blog article, we would like to announce the release of Scala 3.0.0-RC3. Some critical bugs were discovered during the previous release cycle the fixes to which had to be included in 3.0.0 stable. Therefore, we are having RC3 to give the community time to test our fixes.
+
+This release also impacts the release date for 3.0.0 stable. 3.0.0 stable will get a green light if no critical bugs are discovered until May 12th. Otherwise we may need to have an RC4.
+
+<!--more-->
+
+# Bug fixes included
+- Backport: Fix type test for trait parameter arguments [#12066](https://github.com/lampepfl/dotty/pull/12066)
+- Backport: Set file filter correctly [#12119](https://github.com/lampepfl/dotty/pull/12119)
+- Backport: Provide mirror support after inlining [#12079](https://github.com/lampepfl/dotty/pull/12079)
+- Fix type test for trait parameter arguments [#12064](https://github.com/lampepfl/dotty/pull/12064)
```

Comment:
This points to a PR that was closed without merging. It should probably not be there.